pkg/controllers/workspace_controller.go

```diff
@@ -423,6 +435,54 @@ func (c *WorkspaceReconciler) ensureService(ctx context.Context, wObj *kaitov1al
 	return nil
 }
 
+func (c *WorkspaceReconciler) applyTuning(ctx context.Context, wObj *kaitov1alpha1.Workspace) error {
+	var err error
+	func() {
+		if wObj.Tuning.Preset != nil {
+			presetName := string(wObj.Tuning.Preset.Name)
+			model := plugin.KaitoModelRegister.MustGet(presetName)
+
+			trainingParam := model.GetTrainingParameters()
+
+			existingObj := &appsv1.Deployment{}
```

Comment:
This cannot be deployment. It needs to be Job.

Reply:
Done